lib/dal/src/func/argument.rs

```diff
@@ -62,7 +62,16 @@ pub enum FuncArgumentError {
 
 type FuncArgumentResult<T> = Result<T, FuncArgumentError>;
 
-#[remain::sorted]
+/// This type gathers all the kinds of things we will store in the
```

Comment:
This comment is a little wrong since it talks about CasValue and the macros. We just want to say: don't reorder, only add to the bottom, this goes into the content store

Reply:
fixed!